CodeApp/monaco-textmate.bundle/index.html

```diff
@@ -9,12 +9,8 @@
     <meta charset="UTF-8" />
   </head>
   <style>
-    @font-face {
```

Comment:
Fira Code import is needed for the font to work. You can test it by selecting Fira Code as the editor font in the settings menu.

Reply:
Ref: https://github.com/thebaselab/codeapp/issues/439

Reply:
It'd be great if we can find a way to dynamically load fonts into WKWebView but so far I haven't found one.

Reply:
Returned deleted code. I will investigate this issue.